elements/lisk-tree/src/merkle_tree/merkle_tree.ts

```diff
@@ -512,25 +403,74 @@ export class MerkleTree {
 				// if layer below is odd numbered, push left child
 				currentLayerSize = this._size >> (currentLayer - 1);
 				if (currentLayerSize % 2 === 1) {
-					const leftNode = await this.getNode(currentNode.leftHash);
+					const leftNode = await this._getNode(currentNode.leftHash);
 					appendPath.push(leftNode);
 				}
 
 				// go to right child
-				currentNode = await this.getNode(currentNode.rightHash);
+				currentNode = await this._getNode(currentNode.rightHash);
 			}
 		}
 		// Append path should be from bottom
 		return appendPath.reverse();
 	}
 
+	private async _getIndices(queryHashes: ReadonlyArray<Buffer>): Promise<number[]> {
+		const idxs = [];
+		const height = this._getHeight();
+		for (const query of queryHashes) {
+			try {
+				const node = await this._getNode(query);
+				idxs.push(toIndex(node.nodeIndex, node.layerIndex, height));
+			} catch (error) {
+				idxs.push(0);
+			}
+		}
+		return idxs;
+	}
+
+	private async _getAppendPathHashes(): Promise<Buffer[]> {
```

Comment:
this is just changing to private function, and by changing, the location within the file needed to be changed. However, the content is exactly the same